File: KeeprVande/Models/Vault.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KeeprVande.Models;

public class Vault : RepoItem<int>
{
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public bool? IsPrivate { get; set; }
  public string CreatorId { get; set; }
}

File: KeeprVande.client/src/models/Vault.js
export class Vault {
  constructor(data) {
    this.id = data.id
    this.name = data.name
    this.img = data.img
    this.description = data.description
    this.isPrivate = data.isPrivate
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
    this.creator = data.creator
    this.creatorId = data.creatorId
  }
}